about.md

```diff
@@ -6,294 +6,48 @@ title: About MDAnalysis
 <img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
 style="float: right" alt="MDAnalysis" width="30%"/>
 
-## Mission
+MDAnalysis is an open-source Python library for analyzing molecular dynamics simulations and biomolecular structures. It enables high-performance, reproducible computational studies, supporting researchers across academia and industry.
 
-The MDAnalysis [Community](#community) is interested in all facets of **working
-with data in the computational molecular sciences**. We welcome everyone. **We all follow our
-[Code of Conduct] and strive to create an environment that is welcoming to
-all.** Our primary purpose is to produce software that scientists in academia and
-industry will trust to use in their research.
+Learn more about our mission on the [Mission]({{ site.baseurl }}/pages/mission/) page.
```

Comment:
I don't like "Learn more" as it makes it like something optional. With the Mission, we should be clear.

The [Mission](mission.md) of the MDAnalysis Project is to ... (summarize).